RecoEcal/EgammaClusterProducers/python/reducedRecHitsSequence_cff.py

```diff
@@ -151,26 +151,29 @@
 #selected digis
 from RecoEcal.EgammaClusterProducers.ecalDigiSelector_cff import *
 
-reducedEcalRecHitsSequence = cms.Sequence(interestingEcalDetIdEB*interestingEcalDetIdEBU*
-                                          interestingEcalDetIdEE*
-                                          interestingEcalDetIdPFEB*interestingEcalDetIdPFEE*interestingEcalDetIdPFES*
-                                          interestingEcalDetIdOOTPFEB*interestingEcalDetIdOOTPFEE*interestingEcalDetIdOOTPFES*
-                                          interestingEcalDetIdRefinedEB*interestingEcalDetIdRefinedEE*interestingEcalDetIdRefinedES*
-                                          interestingTrackEcalDetIds*
-                                          reducedEcalRecHitsEB*
-                                          reducedEcalRecHitsEE*
-                                          seldigis*
+reducedEcalRecHitsTask = cms.Task(interestingEcalDetIdEB,interestingEcalDetIdEBU,
+                                          interestingEcalDetIdEE,
+                                          interestingEcalDetIdPFEB,interestingEcalDetIdPFEE,interestingEcalDetIdPFES,
+                                          interestingEcalDetIdOOTPFEB,interestingEcalDetIdOOTPFEE,interestingEcalDetIdOOTPFES,
+                                          interestingEcalDetIdRefinedEB,interestingEcalDetIdRefinedEE,interestingEcalDetIdRefinedES,
+                                          interestingTrackEcalDetIds,
+                                          reducedEcalRecHitsEB,
+                                          reducedEcalRecHitsEE,
                                           reducedEcalRecHitsES)
+reducedEcalRecHitsSequence = cms.Sequence(reducedEcalRecHitsTask)
+reducedEcalRecHitsSequence += seldigis
```

Comment:
Why not make a seldigisTask in the cff defining seldigis and include that in the reducedEcalRecHitsTask? And the same thing in other spots with seldigis?